src/main/steps/applicant1/equality/createToken.ts

```diff
@@ -0,0 +1,32 @@
+import crypto from 'crypto';
+
+import { Logger } from '@hmcts/nodejs-logging';
+import config from 'config';
+
+const logger = Logger.getLogger('createToken');
+
+const algorithm = 'aes-256-gcm';
+const bufferSize = 16;
+const iv = Buffer.alloc(bufferSize, 0); // Initialization vector.
+const keyLen = 32;
+
+export const createToken = (params: Record<string, unknown>): string => {
+  const tokenKey: string = config.get('services.equalityAndDiversity.tokenKey');
+  let encrypted = '';
+
+  if (tokenKey) {
```

Comment:
do you need this check? is there ever a situation where the tokenKey is not set?

Reply:
Only if there is an issue getting it from the config but I am happy to remove this conditional